assets/js/ajax-filters.js

```diff
@@ -255,26 +262,67 @@ jQuery( document ).ready( function ( $ ) {
 		$supports_html5_history = false;
 	}
 
-	var location = document.location.href.split('#')[0];
-
 	function job_manager_store_state( target, page ) {
+		// changed this to ? because this is the standard query string identifier in php
+		// and changed to window.location for better cross browser support
+		// and moved into the function so it rechecks everytime the function is called
+		var location = window.location.href.split('?')[0];
+		var query = window.location.href.split('?')[1];
+		query = query.split('&');
```

Comment:
So I can either write a url query setter and getter myself to surgically alter our specific queries whilst leaving the rest of the URI string intact, or I can use this plugin [https://github.com/medialize/URI.js](https://github.com/medialize/URI.js) but perhaps that adds unneccessary overhead? What do you think @mikejolley ??